Dockerfile

```diff
@@ -1,28 +1,32 @@
 # Client
-FROM public.ecr.aws/bitnami/node:14.17.3 AS client
+FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS client
 
 # Build client
-WORKDIR /client
+# RUN apk add --no-cache git python g++ make
+USER root
+WORKDIR /opt/app-root/src/app/client
 COPY client/package*.json ./
 RUN npm set progress=false && npm ci --no-cache
 COPY client/. .
 RUN INLINE_RUNTIME_CHUNK=false npm run build
-
+USER 1001
```

Comment:
I haven't seen this before do you have a link to what this means?

Reply:
USER 1001 is default user of rhel container, I find this in many CSI-Lab project but let me find appropriate document.

Reply:
[Link](https://developers.redhat.com/blog/2020/10/26/adapting-docker-and-kubernetes-containers-to-run-on-red-hat-openshift-container-platform#)